ambari-infra-manager/pom.xml

```diff
@@ -383,6 +391,14 @@
           <groupId>org.eclipse.jetty</groupId>
           <artifactId>jetty-security</artifactId>
         </exclusion>
+        <exclusion>
+          <groupId>org.bouncycastle</groupId>
+          <artifactId>bcprov-jdk15on</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>com.fasterxml.jackson.core</groupId>
+          <artifactId>jackson-databind</artifactId>
+        </exclusion>
```

Comment:
Why we exclude `jackson-databind` here ? Since original commit didn't do this.

Reply:
I have check, yes we do not need this one.
It was not part of the original commit
I will update the PR